cg-spec/editors_draft.html

```diff
@@ -941,7 +941,206 @@ <h2>SPARQL-star Query Results XML Format</h2>
+             data-content-type="application/x-sparql-star-query"
+             class="nohighlight example"
+        >
+          <!--
+          PREFIX : <http://www.example.org/>
+
+          INSERT DATA {
+            :bob :age 23 .
+            :alice :claims << :bob :age 23 >> .
+          }
+          -->
+        </pre>
+
+      </section>
+
+      <section>
+        <h2>DELETE DATA</h2>
+
+        <p>The <a data-cite="SPARQL11-UPDATE#deleteData">DELETE DATA</a> operation can be used to remove a given set of triples from the graph store. In the context of SPARQL-star Update, this may include removing nested RDF-star triples as demonstrated in the following example.</p>
+
+        <pre data-transform="updateExample"
+             data-content-type="application/x-sparql-star-query"
+             class="nohighlight example"
+        >
+          <!--
+          PREFIX : <http://www.example.org/>
+
+          DELETE DATA {
+            :alice :claims << :bob :age 23 >> .
+          }
+          -->
+        </pre>
+
+        <p>After executing this DELETE DATA operation over a graph store with a default graph that contains the given nested triple, the graph will not contain this triple any longer. If the graph did not contain that nested triple in the first place, the graph will remain unchanged by the given DELETE DATA operation.</p>
+
+        <p>Notice that deleting triples by using the DELETE DATA operation does not affect any other triples in the corresponding graphs. For instance, for a default graph that contains the triple <code>:bob :age 23</code> as an asserted triple, the DELETE DATA operation given above does <em>not</em> delete this asserted triple. In contrast, the following operation would delete this asserted triple but it would <em>not</em> delete any nested triple that contains the given triple as an embedded triple.</p>
+
+        <pre data-transform="updateExample"
+             data-content-type="application/x-sparql-star-query"
+             class="nohighlight example"
+        >
+          <!--
+          PREFIX : <http://www.example.org/>
+
+          DELETE DATA {
+            :bob :age 23 .
+          }
+          -->
+        </pre>
+
+      </section>
+
+      <section>
+        <h2>DELETE/INSERT</h2>
+
+        <p>The <a data-cite="SPARQL11-UPDATE#deleteInsert">DELETE/INSERT</a> operation can be used to remove or add triples based on variable bindings obtained by evaluating a given WHERE clause. As an example, the following DELETE/INSERT operation replaces all nested triples in which <code>:alice</code> is the subject by nested triples in which <code>:alicia</code> is the subject.</p>
+
+        <pre data-transform="updateExample"
+             data-content-type="application/x-sparql-star-query"
+             class="nohighlight example"
+        >
+          <!--
+          PREFIX : <http://www.example.org/>
+
+          DELETE { :alice  ?pp <<?s ?p ?o>> . }
+          INSERT { :alicia ?pp <<?s ?p ?o>> . }
```

Comment:
Too subtle! Suggest something more clearly different, like ":carol"

Reply:
Good point! ;-)
I have changed it to ':carol' in commit https://github.com/w3c/rdf-star/pull/98/commits/b9aae14f4469298c24bd1d832116e8b8b6651fa6